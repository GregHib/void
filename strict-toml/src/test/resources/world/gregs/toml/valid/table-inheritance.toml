[x.y.z]
alphabet = true

[.a] # inheriting previous table
name = "a"

[.a] # inheriting same table
start = true