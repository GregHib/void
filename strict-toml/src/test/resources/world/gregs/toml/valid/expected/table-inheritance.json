{"x":{"y":{"z":{"a":{"name":"a","start":true},"alphabet":true}}}}